### PR TITLE
Implement remaining HTTP routes

### PR DESCRIPTION
**Pull request by Ariazoox**

### CODE REVIEW

File: cmd/http-rest-server/http-rest-server.go [hunk 1]
```diff
@@ -104,3 +107,171 @@ func getDataBetweenTwoTimes(w http.ResponseWriter, r *http.Request) {
...
+
+	parsedDate, parsedDateErr := time.Parse(time.RFC3339, decodedDate)
+	if parsedDateErr != nil {
+		http.Error(w, "Invalid date format (It should be in ISO format)", http.StatusBadRequest)
+		return
+	}
+
+	startTime := time.Date(parsedDate.Year(), parsedDate.Month(), parsedDate.Day(), 0, 0, 0, 0, parsedDate.Location())
+	endTime := startTime.Add(24 * time.Hour).Add(-time.Second)
+	fmt.Println(startTime)
```
Comment (soufiane-ezzemany) [inline cmd/http-rest-server/http-rest-server.go:212]: No need for this print

Comment (soufiane-ezzemany) [inline cmd/http-rest-server/http-rest-server.go:212]: remove this line and the one after

**Review by soufiane-ezzemany — CHANGES_REQUESTED** [review]
remove the fmt.Println

**Review by yousoumar — APPROVED** [review]

**Review by soufiane-ezzemany — APPROVED** [review]
LGTM

**Pull request merged**